app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt

```diff
@@ -162,10 +223,119 @@ class TalkTopicActivity : BaseActivity() {
         }
     }
 
+    inner internal class ReplyTextWatcher: TextWatcher {
```

Comment:
`inner` can be removed.

Reply:
it cannot.

Reply:
You're correct. Would it be possible to add `@Suppress("RedundantInnerClassModifier")` for it?
```
    @Suppress("RedundantInnerClassModifier")
    internal inner class ReplyTextWatcher: TextWatcher {
```

Reply:
yep